appveyor.yml

```diff
@@ -24,6 +24,10 @@ branches:
 # https://www.appveyor.com/docs/installed-software/#ruby
 environment:
   matrix:
+    - ruby_version: "24-x64"
+      devkit: C:\Ruby23-x64\DevKit
```

Comment:
`devkit:` section is meaningless for RubyInstaller2 version of  Ruby2.4.
We can enable C extension building via `ridk.cmd` when using RubyInstaller2's Ruby.
ref: https://github.com/oneclick/rubyinstaller2#the-ridk-command